packages/11ty/.env.example

```
@@ -1,3 +1,4 @@
 ELEVENTY_ENV=development
 ELEVENTY_OUTPUT_DIR=_site
+OUTPUT=html
```

Does it make this variable more explicit to add a `QUIRE_` prefix?
```suggestion
QUIRE_OUTPUT=html
```